test/serializer/optimized-functions/ConditionallyLeakedBinding.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    function f(c, g) {
+        let x = 23;
```

Comment:
There is no object here, what does this test?

Reply:
I tried a slew of different tests to break your unconditional leak declarative bindings patch. This one I decided to keep (don't quite remember why). If you think it's covered by something else, I can drop it.